### DISCUSSION TITLE
Maybe Bug?

### DISCUSSION THREAD

**Discussion by joshwegener** [Laravel-Backpack/community-forum#876] [Q&A (Help)] [answered]
Hello,

I'm having some issues with images. I'm just starting with Backpack and I'm very new to it.

My CRUD is setup like so:
```
$this->crud->addField([
    'name' => 'logo',
    'label' => 'Logo',
    'type' => 'image',
    'crop' => true,
    'aspect_ratio' => 0,
    'withFiles' => true,
    'hint' => 'The logo of the site',
]);
```

On the list view, I add a column for image, but it does not display. I noticed it's looking for image at `https://mysite.com/3gKOghVQUil5nfdebN2VoE6b3a2SJsPPu1DspTYf.png` but the image is stored in `https://mysite.com/storage/3gKOghVQUil5nfdebN2VoE6b3a2SJsPPu1DspTYf.png`

Edit view works fine and shows image correctly. 

Show view using image to display uses the wrong path as well.

Is this a bug or I’m doing something wrong??

```
    // Same as setupListOperation()
    public function setupShowOperation()
    {
        $this->crud->addColumn([
            'name' => 'logo',
            'label' => 'Logo',
            'type' => 'image',
            'height' => '40px',
            'width' => '40px',
        ]);
        // other fields
    }
    ```

**Comment by jcastroa87** [comment, marked as answer]
Hello @joshwegener 

In the column try to add a prefix

```
$this->crud->addColumn([
            'name' => 'logo',
            'label' => 'Logo',
            'type' => 'image',
            'height' => '40px',
            'width' => '40px',
            'prefix' => 'storage/',
        ]);
```

Let me know if that works for you.

Cheers.

**Reply by joshwegener** [reply]
This did fix the issue. Thank you.

**Comment by jcastroa87** [comment]
Great, I will close the issue; feel free to re-open it if needed or create a new one.

Cheers.
